site/templates/film/item.jet

```diff
@@ -18,21 +18,39 @@
 {{end}}
 
 {{block body()}}
+
+{{ sponsorText := film.CustomFields.GetString("sponsor_image_text_film", "") }}
+{{ sponsorLink := film.CustomFields.GetString("sponsor_image_link_film", "") }}
+{{ sponsorLinkIsExternal := film.CustomFields.GetBool("sponsor_image_link_external_film", false) }}
+{{ sponsorImage := film.ImageMap["Sponsor"] }}
+
   <main id="main" class="page page-film meta-detail meta-detail-film">
     <div class="meta-detail-bg">
       <div class="right-gradient"></div>
       <s72-image src="{{film.ImageMap["Background"]}}" alt="" class="meta-detail-bg-img"></s72-image>
     </div>
     <div class="container">
       <div class="meta-detail-main">
-        {{if config("default_image_type") == "portrait"}}
-            <div class="poster poster-portrait">
-              <s72-availability-status data-slug="{{film.Slug}}"></s72-availability-status>
-              <s72-image src="{{film.ImageMap["Classification"]}}" alt="" class="classification-image">
-              <s72-image src="{{film.ImageMap["Portrait"]}}" alt="{{film.Title}}" class="poster poster-image poster-image-portrait"></s72-image>
-
+        <div class="sponsor-poster">
```

Comment:
I'm not liking this class name, I read it as the poster for the sponsor. I get that you want a wrapper around both the sponsor and poster but if there isn't a sponsor, it's a bit misleading. How about `poster-wrapper` or something?

```suggestion
        <div class="sponsor-wrapper">
```

Reply:
Hmm, this has dropped out of order, was before the other one!

Reply:
- changed to  .sponsor-wrapper